tests/bluetooth/audio/bap_base/prj.conf

```diff
@@ -2,9 +2,13 @@ CONFIG_ZTEST=y
 
 CONFIG_BT=y
 CONFIG_BT_AUDIO=y
+CONFIG_BT_PERIPHERAL=y
```

Comment:
Why do you have to enable peripheral?
I can't see any dependence between CONFIG_BT_BAP_SCAN_DELEGATOR and CONFIG_BT_PERIPHERAL. It do is required for other roles in 'Kconfig.bap' though

Reply:
You are right. I believe it is because I originally added `BT_BAP_BROADCAST_SINK` instead of `BT_BAP_SCAN_DELEGATOR`

That being said, this may need to change in the future, as `BT_BAP_SCAN_DELEGATOR` requires support for GATT which in turn require support for `BT_CONN`, but that will be a separate PR